test/unit/logger-mixin.test.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {LoggerMixin} from '../..';
+import {Application, Component} from '@loopback/core';
+import {Constructor} from '@loopback/context';
+
+describe('LoggerMixin', () => {
```

Comment:
Please move this file to `test/unit/mixins/logger.mixin.test.ts`. Related discussions: https://github.com/strongloop/loopback-next-extension-starter/pull/6#discussion_r142387171 and https://github.com/strongloop/loopback-next-extension-starter/pull/6#discussion_r142387402